src/main/java/pl/mantiscrab/budgetr/config/H2Config.java

```diff
@@ -0,0 +1,17 @@
+package pl.mantiscrab.budgetr.config;
+
+import org.h2.tools.Server;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.sql.SQLException;
+
+@Configuration
+@Profile({"dev", "test"})
+class H2Config {
```

Comment:
It's fully fine to have this canfig in here bu I thought that autoconfiguration is doing exactly the same

Reply:
It doesn't. For further explanations: https://www.youtube.com/watch?v=HlvF2TlA1Yo
Tl;dr : without it IntelliJ creates it's own instance of H2 database while connecting, so you don't have access to "application's" database